indico/modules/receipts/default_templates/attendance/template.html

```diff
@@ -0,0 +1,95 @@
+{% set personal_data = registration.personal_data %}
+{% set today = now_utc() %}
+
+<h1>Certificate of Attendance</h1>
```

Comment:
should we make this configurable as well (e.g. for people who want it in a different language)? currently it's not configurable for the receipt either though...